README.md

```diff
@@ -20,9 +20,22 @@ This package does not aim to provide comprehensive production quality plotting.
 
 ### Installation
 
+Install InteractiveViz.jl with one of the Makie backends that support interactivity (GLMakie or WebGLMakie).
+
 ```julia
 julia>]
-pkg> add InteractiveViz
+pkg> add InteractiveViz GLMakie
+```
+
+If more than one Makie backend is available, switch between them in the usual way.
+
+```julia
+using GLMakie
+
+GLMakie.activate!()
```

Comment:
```suggestion
using GLMakie
GLMakie.activate!()
```

Reply:
Minor README formatting

Reply:
Ah, I follow a style convention which mandates a space between using/imports and actual use, but happy to adjust 😄

Reply:
In that case, the next one importing `WGLMakie` needs a blank line. I took cue from there :-)

In general, I like spaces after imports. But in the README, in the interest of space, I might prefer without.

Reply:
Ah, yes sorry - poor quality control on my part!